src/NuGet.Core/NuGet.CommandLine.XPlat/GlobalSuppressions.cs

```diff
@@ -30,3 +30,7 @@
 [assembly: SuppressMessage("Build", "CA1062:In externally visible method 'Task<int> RemovePackageReferenceCommandRunner.ExecuteCommand(PackageReferenceArgs packageReferenceArgs, MSBuildAPIUtility msBuild)', validate parameter 'msBuild' is non-null before using it. If appropriate, throw an ArgumentNullException when the argument is null or add a Code Contract precondition asserting non-null argument.", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.CommandLine.XPlat.RemovePackageReferenceCommandRunner.ExecuteCommand(NuGet.CommandLine.XPlat.PackageReferenceArgs,NuGet.CommandLine.XPlat.MSBuildAPIUtility)~System.Threading.Tasks.Task{System.Int32}")]
 [assembly: SuppressMessage("Build", "CA1819:Properties should not return arrays", Justification = "<Pending>", Scope = "member", Target = "~P:NuGet.CommandLine.XPlat.PackageReferenceArgs.Frameworks")]
 [assembly: SuppressMessage("Build", "CA1819:Properties should not return arrays", Justification = "<Pending>", Scope = "member", Target = "~P:NuGet.CommandLine.XPlat.PackageReferenceArgs.Sources")]
+[assembly: SuppressMessage("Globalization", "CA1305:Specify IFormatProvider", Justification = "<Pending>", Scope = "member", Target = "~M:Microsoft.Extensions.CommandLineUtils.CommandLineApplication.GetHelpText(System.String)~System.String")]
```

Comment:
You can say why those suppressions are needed

```suggestion
[assembly: SuppressMessage("Globalization", "CA1305:Specify IFormatProvider", Justification = "External source file injected via NuGet contentFiles. Cannot be modified", Scope = "member", Target = "~M:Microsoft.Extensions.CommandLineUtils.CommandLineApplication.GetHelpText(System.String)~System.String")]
```